scripts/test_schemas.sh

```diff
@@ -5,21 +5,54 @@ validator=$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)
 
 sleep 3
 
-echo "Testing Schemas"
-
 exit=0
 
-for schema in $(find $1 -name '*.json');
-do
-echo $schema
-result=$(curl -X POST -H "Content-Type: text/plain" --data "$(cat $schema)" http://localhost:5001/validate 2>/dev/null)
-if [ "$result" != "{}" ] && [ "$result" != "" ]; then
-echo "---Schema Failed Validation---"
-echo "Error: [$result]"
-exit=1
-fi
+for i in {1..3}; do
+    response=$(curl -so /dev/null -w '%{http_code}' http://localhost:5001/status)
+    if [ "$response" == "200" ]; then
+        echo "---Testing Schemas---"
+        for schema in $(find "$1" -name '*.json'); do
```

Comment:
Rather than nesting this `for` you could invert the previous `if` and exit on 3rd fail?